### PR TITLE
Cards structure

### PR DESCRIPTION
**Pull request by SteRed**
we reorganized the structure of the button cards

### CODE REVIEW

File: components/button-cards/category-cards/category-card-dark/category-card-dark.config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 status: 'wip'
-notes: 'Overview of Category Cards'
+notes: 'Overview of available colors'
```
Comment (mmussmann) [inline components/button-cards/category-cards/category-card-dark/category-card-dark.config.yml]: hm doesn't seem right?

Comment (SteRed) [reply components/button-cards/category-cards/category-card-dark/category-card-dark.config.yml]: that was not part of what we were doing but we fixed it.

File: components/button-cards/contact-card/contact-card.scss [hunk 2]
```diff
@@ -180,3 +174,22 @@
     z-index: 1;
   }
 }
+
+@media (min-width: 769px) and (max-width: 992px) {
```
Comment (ms-Danny) [inline components/button-cards/contact-card/contact-card.scss]: I would recommend to move the media query to the relevant element. In this case you have to scroll multiple times in the file to get the complete overview over an element. Which i think is not ideal

**Comment by SteRed** [comment]
Hi Danny

I have been working with media query inside class definitions and I ended up having them at the bottom as it is much more readable and easier to maintain. When you have many media queries then it start to be the mess especially if the implementation has been collapsed.

When it is in the end, even though it is collapsed the first line will represent the media query. But yes you have to scroll a little bit but in the end it is cleaner.

Best regards


Stephan Redon
Senior Software Engineer
Web Development
Mobile: +45 50 842 960

[cid:image010.jpg@01D56327.6A83BAF0]<https://linkedin.com/company/milestone-systems>[cid:image011.jpg@01D56327.6A83BAF0]<https://twitter.com/milestonesys>[cid:image012.jpg@01D56327.6A83BAF0]<https://www.youtube.com/user/Milestonesys>[cid:image013.jpg@01D56327.6A83BAF0]<https://www.facebook.com/milestonesys/>[cid:image014.png@01D56327.6A83BAF0]<https://www.instagram.com/milestonesystems/>

[cid:image015.png@01D56327.6A83BAF0]<https://www.milestonesys.com/>

**Pull request merged**